modules/phase_field/tests/phase_field_kernels/CoupledAllenCahn.i

```diff
@@ -0,0 +1,122 @@
+#
+# Test the parsed function free enery Allen-Cahn Bulk kernel
+#
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 20
+  ny = 20
+  nz = 0
+  xmin = 0
+  xmax = 50
+  ymin = 0
+  ymax = 50
+  zmin = 0
+  zmax = 50
+  elem_type = QUAD4
+
+  uniform_refine = 1
```

Comment:
... and I typed that so fast that I failed to follow normal grammar rules in my response to your incorrect grammar usage.